pixeltable/catalog/table.py

```diff
@@ -168,8 +184,14 @@ def describe(self) -> None:
         except NameError:
             print(self.__repr__())
 
+    # TODO: Display comments in _repr_html()
```

Comment:
We should follow up on that pretty quickly, _repr_html() is what runs in a notebook.

Reply:
Sure. I just wasn't sure how to manipulate the Styler properly etc. I'd propose leaving this as a TODO for now, since nothing is actually broken (and you can still see the comment via t.comment())